code/obj/item/kendo.dm

```diff
@@ -147,16 +151,17 @@
                 var/parry_block = S.parry_block_check(attacker,defender)
                 if((parry_block == 1) || (parry_block == 2))
                     attacker.do_disorient((attacker.equipped().stamina_damage),0,0,0,0,1,null)
-                    return
-                else if((attacker.a_intent=="disarm") && prob(20) && defender.equipped())
-                    var/obj/item/I = defender.equipped()
-                    defender.u_equip(I)
-                    I.set_loc(defender.loc)
-                    var/target_turf = get_offset_target_turf(I.loc,rand(5)-rand(5),rand(5)-rand(5))
-                    SPAWN_DBG(1 DECI SECOND)
-                    I.throw_at(target_turf,3,1)
-                    defender.show_text("<b>[attacker] knocks the [I] right out of your hands!</b>","red")
-                    attacker.show_text("<b>You knock the [I] right out of [defender]'s hands!</b>","green")
+                    return //stops damage if parried or blocked, if not, itll check for a disarm
+
+            if((attacker.a_intent=="disarm") && prob(20) && defender.equipped())
+                var/obj/item/I = defender.equipped()
+                defender.u_equip(I)
+                I.set_loc(defender.loc)
+                var/target_turf = get_offset_target_turf(I.loc,rand(5)-rand(5),rand(5)-rand(5))
+                SPAWN_DBG(1 DECI SECOND)
+                I.throw_at(target_turf,3,1)
```

Comment:
missing indent after spawn?

Reply:
I thought that might be an issue, but ill fix it soon

Reply:
fixy fixed～

Reply:
Don't you *want* the throw_at to be in a spawn though? Otherwise it will hold up the proc (throw_at sleeps)

Reply:
true...i readded it :P